src/App.tsx

```diff
@@ -1,61 +1,152 @@
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import users from './api/users';
+import todosFromServer from './api/todos';
+
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+
+const findUserById = (userId: number): User | null => (
+  users.find(user => user.id === userId) || null
+);
+
+const usersTodoList: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: findUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [todos, setTodos] = useState(usersTodoList);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [completed, setCompleted] = useState(false);
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+
+  const handleError = () => {
```

Comment:
The function handles two errors.
```suggestion
  const handleErrors = () => {
```